homeassistant/components/updater/__init__.py

```diff
@@ -95,15 +96,24 @@ def _load_uuid(hass, filename=UPDATER_UUID_FILE):
             newest = hass.components.hassio.get_homeassistant_version()
 
         # Validate version
+        update_available = False
         if StrictVersion(newest) > StrictVersion(current_version):
             _LOGGER.info("The latest available version is %s", newest)
-            hass.states.async_set(
-                ENTITY_ID, newest, {ATTR_FRIENDLY_NAME: 'Update Available',
-                                    ATTR_RELEASE_NOTES: releasenotes}
-            )
+            update_available = True
         elif StrictVersion(newest) == StrictVersion(current_version):
             _LOGGER.info(
                 "You are on the latest version (%s) of Home Assistant", newest)
+        elif StrictVersion(newest) < StrictVersion(current_version):
+            _LOGGER.warning(
+                "You are before the latest version (%s) of Home Assistant",
```

Comment:
Can we use another word than "before"? I think it is vague what we mean with that.

Reply:
I've changed the text. It is now "Local version is newer than the latest version (%s)", newest). Some linux package managers (pacman - Arch Linux) use a similar sentence.
https://bbs.archlinux.org/viewtopic.php?id=128502